app/templates/items/new.hbs

```diff
@@ -161,6 +161,18 @@
         </div>
       {{/if}}
     </div>
+    <div class="row">
+      <div class="small-3 columns item-label">
+        Saleable
+      </div>
+      <div class="small-3 columns publish">
+        {{input type="checkbox"
+        checked=isSaleable}}
+      </div>
+      <div class="small-9 columns">
```

Comment:
Minor thing.. you can remove this div. Because the next div is having `row` class it is anyhow going to take new line. Or next thing you can do is use  `offsett` class in second div with class `publish`.